### ISSUE TITLE
Document database considerations

### ISSUE THREAD

**Issue by brycelednar** [hackforla/HomeUniteUs#492]
Document considerations for data hosting and relevant privacy regulations

**Comment by brycelednar** [comment]
Created this [Google doc](https://docs.google.com/document/d/1kohmbravTv5AS96AFzwyO3_ZnoBhoDoOyiS7-RfsWYA/edit) for documentation

**Comment by brycelednar** [comment]
Hey @ju1es , picking up the HMIS conversation from earlier, I created this issue to track database considerations and assumptions. 

One of the assumptions in the doc is that Host Home orgs want to integrate with HMIS by default. This would make sense because Host Homes need to integrate to [receive funding from HUD](https://www.hudexchange.info/programs/hmis/hmis-requirements/) and for data security + sharing purposes. 

I have come across articles expressing concern about HMIS integration, specifically regarding [data privacy](https://www.aclu.org/other/comments-arguing-against-hmis) and positioning host homes as ["non-institutional" services](https://imprintnews.org/opinion/creative-community-responses-youth-homelessness/20744), "outside the system", and [inconsistent usage of HMIS](https://static1.squarespace.com/static/553bd8dfe4b06d949518334e/t/5991db9a914e6be95f116e70/1502731168374/Commerce-Host-Homes-Report-2017.pdf) across host home programs. 

tldr my take on the way forward here is to 1) For Engineering: continue learning about how to integrate with HMIS as this is the expected path, and 2) For PM: get validation from some host home orgs to get a gutcheck on any concerns to be mindful of with respect to HMIS.

Let me know what you think, 

Bryce

**Comment by ju1es** [comment]
@brycelednar , thank you for creating this! before jumping into researching more about HMIS, I want to verify that we're actually required to integrate. I just read ```Service providers that receive Federal funding and some State funding are required to participate in local HMIS``` [here](https://bcsh.ca.gov/calich/hdis/faq.html). 

Same thing applies to the other privacy frameworks in the doc. I'm thinking that once we know which frameworks we need to use, you get validation from stakeholders, then I can go ahead and look up the architectural and design requirements for each framework. We'd also have a clear goal of how we can be privacy compliant? This is all new to me but just my initial thoughts.
